src/test/resources/application-test.yml

```
@@ -0,0 +1,41 @@
+
+cookie:
+  domain: localhost
+
+openai:
+  api-key: test
+  base-url: http://127.0.0.1:9999
+  prompt:
+    id: test
+    version: 1
+aws:
+  enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)